packages/eas-cli/src/commands/build/index.ts

```diff
@@ -135,6 +142,12 @@ export default class Build extends EasCommand {
       );
       Log.newLine();
     }
+    if (
+      flags['resource-class'] &&
```

Comment:
do you need to validate that? I thought that ocliff is handling that out of the box?

Reply:
ah I didn't realize that was the case, I'll remove this